README.md

```diff
@@ -1,54 +1,149 @@
+
+TODO it's not --release
+
+If your distro isn't listed above, `libblip_buf` probably isn't in your package repos. You can easily [build it yourself](https://gitlab.com/TASVideos/libblip_buf/blob/unified/README.md).
+
+Once built, see the *Installing* section above, substituting the repo's `output` folder for the download. If your distro isn't listed there, you might get an "Unknown distro" warning in the terminal, and BizHawk may not open or may show the missing dependencies dialog. You may need to add your distro to the case statement in the script, setting `libpath` to the location of `d3dx9_43.dll.so` (please do share if you get it working).
+
+### Windows 7/8.1/10
+
+TODO
+
+[Compiling](http://tasvideos.org/Bizhawk/Compiling.html)
+
+## Usage (casual)
+
+TODO
+
+## Usage (advanced)
+
+TODO
+
+[Commandline](http://tasvideos.org/Bizhawk/CommandLine.html)
+
+[CompactDiscInfoDump](http://tasvideos.org/Bizhawk/CompactDiscInfoDump.html)
+
+[Rerecording](http://tasvideos.org/Bizhawk/Rerecording.html)
+
+[TAS movie file format](http://tasvideos.org/Bizhawk/TASFormat.html)
+
+## Support and troubleshooting
+
+A short [FAQ](http://tasvideos.org/Bizhawk/FAQ.html) is provided on the [BizHawk wiki](http://tasvideos.org/Bizhawk.html). If your problem is one of the many not answered there, and you can't find it in the [issue tracker search](https://github.com/TASVideos/BizHawk/issues?q=is%3Aissue+ISSUE_KEYWORDS), check the [BizHawk forum](http://tasvideos.org/forum/viewforum.php?f=64) at tasvideos.org, or ask on IRC:
+
+[![IRC | chat.freenode.net #bizhawk](https://img.shields.io/badge/IRC-chat.freenode.net_%23bizhawk-steelblue.svg?style=flat)](ircs://chat.freenode.net:6697/bizhawk)
+[![Matrix (IRC bridge) | #freenode_#bizhawk:matrix.org](https://img.shields.io/badge/Matrix_(IRC_bridge)-%23freenode__%23bizhawk:matrix.org-mediumpurple.svg?logo=matrix&style=flat)](https://matrix.to/#/#freenode_#bizhawk:matrix.org)
+[![freenode webchat | #bizhawk](https://img.shields.io/badge/freenode_webchat-%23bizhawk-lightcoral.svg?style=flat)](http://webchat.freenode.net/?channels=bizhawk)
+
+## Contributing
+
+BizHawk is Open Source Software, so you're free to modify it however you please, and if you do, we invite you to share! Under the MIT license, this is *optional*, just be careful with reusing cores as some have copyleft licenses.
+
+If you've already made changes, the best way to contribute is with a Pull Request here on GitHub. If you're looking for something to do, check the issue tracker here on GitHub:
+
+[![GitHub open issues counter](https://img.shields.io/github/issues-raw/TASVideos/BizHawk.svg?logo=github&style=flat)](https://github.com/TASVideos/BizHawk/issues)
+
+It's a good idea to check if anyone is already working on an issue by asking on IRC (see *Support* above). And of course, open a "feature request" issue before adding features. No point making something that no-one's going to use.
```

Comment:
It's said here, but move it above so it's obvious making an issue is the default and preferred option.

Reply:
this is the same criticism twice? if you mean we should tell users to open issues for bugs, I'd forgotten that. it's in now